src/components/exchange/ExchangeNotch.tsx

```diff
@@ -1,36 +1,30 @@
 import React from 'react';
 import FastImage from 'react-native-fast-image';
 import { useTheme } from '../../theme/ThemeContext';
-import { Row } from '../layout';
 import ExchangeNotchLeft from '@rainbow-me/assets/exchangeNotchLeft.png';
 import ExchangeNotchLeftDark from '@rainbow-me/assets/exchangeNotchLeftDark.png';
 import ExchangeNotchMiddle from '@rainbow-me/assets/exchangeNotchMiddle.png';
 import ExchangeNotchMiddleDark from '@rainbow-me/assets/exchangeNotchMiddleDark.png';
 import ExchangeNotchRight from '@rainbow-me/assets/exchangeNotchRight.png';
 import ExchangeNotchRightDark from '@rainbow-me/assets/exchangeNotchRightDark.png';
+import { Box } from '@rainbow-me/design-system';
 import { useDimensions } from '@rainbow-me/hooks';
 import styled from '@rainbow-me/styled-components';
 
 const notchHeight = 48;
 const notchSideWidth = 78;
 const ANDROID_NOTCH_OFFSET = 8;
 
-const Container = styled(Row).attrs({
-  pointerEvents: 'none',
-})({
-  height: notchHeight,
-  left: 0,
-  position: 'absolute',
-  top: 132,
-});
-
-const NotchMiddle = styled(FastImage).attrs(({ isDarkMode }) => ({
-  resizeMode: FastImage.resizeMode.stretch,
-  source: isDarkMode ? ExchangeNotchMiddleDark : ExchangeNotchMiddle,
-}))({
+const NotchMiddle = styled(FastImage).attrs(
+  ({ isDarkMode }: { isDarkMode: boolean }) => ({
+    resizeMode: FastImage.resizeMode.stretch,
+    source: isDarkMode ? ExchangeNotchMiddleDark : ExchangeNotchMiddle,
+  })
+)({
   height: notchHeight,
   left: android ? -ANDROID_NOTCH_OFFSET : 0,
-  width: ({ deviceWidth }) => deviceWidth - notchSideWidth * 2.11,
+  width: ({ deviceWidth }: { deviceWidth: number }) =>
+    deviceWidth - notchSideWidth * 2.11,
 });
 
 const NotchSide = styled(FastImage)({
```

Comment:
@markdalgleish Are we trying to deprecate the use of `styled()`?

Reply:
i'm not mark but i believe so

Reply:
If that’s the case do we have an api that replaces that functionality, as it feels useful for composability.

Reply:
Eventually, but don't feel like you need to change this if there isn't a clear replacement for it yet.

Reply:
Wonder if we could even replicate `styled-components` by using polymorphism on `Box` for now (e.g. `<Box as={...}>`), then when we have a clear replacement, migration will be trivial.

Before:

```tsx
const NotchSide = styled(FastImage)({
  height: android ? notchHeight + 2 : notchHeight,
  left: android ? -ANDROID_NOTCH_OFFSET : 0,
  width: android ? notchSideWidth + ANDROID_NOTCH_OFFSET : notchSideWidth
});
```

After:

```tsx
const NotchSide = () => 
  <Box 
    as={FastImage}
    height={{ custom: android ? notchHeight + 2 : notchHeight }}
    left={{ custom: android ? -ANDROID_NOTCH_OFFSET : 0 }}
    width={{ custom: android ? notchSideWidth + ANDROID_NOTCH_OFFSET : notchSideWidth }}
  />
```

Thoughts?

Reply:
Yeah I like this idea, what do we estimate the lift of enabling this or is it already part of the Box api?

Reply:
i think this ANDROID_NOTCH_OFFSET is not needed anymore with this components migration

this is how it looks  (android left, ios right)
![image](https://user-images.githubusercontent.com/12115171/187999954-f27ca4d8-9868-4be1-9dfe-1163f084eb82.png)

without it, notch side is visible on android
![image](https://user-images.githubusercontent.com/12115171/188000041-e79c1b25-2477-4e7f-a9c8-10029133d3ce.png)

![image](https://user-images.githubusercontent.com/12115171/188000186-833c9c57-b18e-488f-ae37-9643b8a08acf.png)

Reply:
also noticed shadows aren't working on android for the token symbol and buttons